packages/docs/pages/StatefulTable/StatefulTablePage.tsx

```diff
@@ -223,6 +229,48 @@ const StatefulTablePage = () => {
         />
         {/* jsx-to-string:end */}
       </CodePreview>
+
+      <H2>Usage with search and filters</H2>
```

Comment:
2 comments on this one:
1. Thanks for using the proper heading here (`H2`)! While you're here, could you also update `Usage with drag and drop` and `Usage with filters` to use a `H2` as well?
2. Seems like this example does way more than just `search and filters`. Can we simplify this to just searching? We already have a filters example above.